frontend/src/services/AuthService.ts

```diff
@@ -0,0 +1,56 @@
+import router from '@/router';
+import { reactive, readonly } from 'vue';
+
+// Auth state from localStorage
+const state = reactive({
```

Comment:
You don't necessarily need to use local storage for the logged in user versus just using Vue's global state. 
(I know we used local storage in FOM).